README.md

```diff
@@ -87,8 +87,9 @@ let output = filterTypescript(input, options);
 
 ### How to upgrade `typescript`
 
-1. Update `typescript` in `package.json`
-2. Run `yarn run generate-tsconfig-interface`
-3. Update `vendor/typescript`. `cd vendor/typescript && git fetch --tags && git checkout v[new-version-of-typescript]`
-4. Commit all of the above changes
-5. Run `yarn test`. There may be some changes needed to the tests to accomidate changes in TypeScript.
\ No newline at end of file
+1. Initialize git submodules. `git submodule update --init`
```

Comment:
I added this because I had not initially ran a `git submodule update --init`, and so the `cd vendor/typescript && git checkout v3.9.2` bit wouldn't work at all.